AAI/AAIConnectProfile.md

```diff
@@ -462,23 +462,21 @@ TODO embedded-access-token is this a passport access token or a visa access toke
 
 1.  Passport Issuers are used to package Visas into signed Passports.
 
-2.  Passport Issuers do not need to be a be a OIDC provider, and MAY provide a .well-known endpoint that doesn't conform to the OIDC Discovery specification for ease of finding signing keys.  
+2.  Passport Issuers SHOULD be Brokers.
 
-    1. Passport Issuers MAY be AAI Clients, Clearinghouses, Brokers or any other entity and do not need to be part of the OIDC flow.  
-    
-    2. Passports SHOULD be signed in the same way that Brokers sign access tokens.
-
-3.  Passports themselves are JWTs that contain Visas. Passports use this format [Format](#claims-sent-to-data-holder-by-a-broker-via-token-or-userinfo) as a signed JWT.  
+3.  Passports themselves are JWTs that contain Visas. Passports use [this format](#claims-sent-to-data-holder-by-a-broker-via-token-or-userinfo-endpoint) as a signed JWT.
     
     1. It is RECOMMENDED for Passports to conform to the <https://tools.ietf.org/html/rfc7515> (JWS) Specification.  
+
+    2. Passports MUST be signed with the `RS256` or `ES256` algorithm.
```

Comment:
384 or 512 should be allowed but not required. ES256 ought to be listed first.

Reply:
We're going to leave this language here and send a separate PR to replace the language referencing algorithms that is currently distributed throughout the document with a single section that specifies recommended algorithms and a reference to the RFC8725.